docs/changes/units/16455.api.rst

```diff
@@ -0,0 +1,2 @@
+The ``format.Fits`` formatter class has been renamed to ``format.FITS``.
```

Comment:
I'd suggest: 
```
The private ``astropy.units.format.Fits`` formatter class has been renamed to ``format.FITS``.
The old name is deprecated. Note that this does not affect regular user visible behaviour:
the formatters are selected indirectly via their lower-case name (in this case ``format="fits"``).
This change should have no effect except for those who may have subclassed the formatter.
```

Reply:
I expanded the change log entry, but not quite how you suggested. Mentioning ``format="fits"`` seems to assume the use of the `to_string()` method, but the formats can also be used in an f-string (e.g. `f"{q:fits}"`), in which case there is no explicit `format` argument.